manifests/b/Backblaze/BackblazeBackup/8.0.1.571/Backblaze.BackblazeBackup.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Backblaze.BackblazeBackup
+PackageVersion: 8.0.1.571
+InstallerSwitches:
+  Silent: -nogui -signinaccount why@gmanil.com 0Solomon

```

Comment:
```
  Silent: -nogui -signinaccount why@gmanil.com 0Solomon
```
I'm not sure if this is a good idea.

It's best to wait until the package repository has support for interactive-only installers.